src/internet_identity/src/assets.rs

```diff
@@ -21,6 +21,7 @@ pub enum ContentType {
     ICO,
     WEBP,
     CSS,
+    OCTETSTREAM,
```

Comment:
This is the content-type that the asset canister currently uses if you follow the instructions [here](https://github.com/r-birkner/portal/blob/rjb/custom-domains-docs-v2/docs/developer-docs/production/custom-domain/custom-domain.md#custom-domains-on-the-boundary-nodes). It _should_ not make a difference, if we used a different one, like `text/plain`. However, this would afaik be untested.

Reply:
gotcha, thanks for checking though!